client/src/app/interfaces/mentorship-request.ts

```diff
@@ -0,0 +1,15 @@
+export interface MentorshipRequestCreateDTO {
```

Comment:
nit: not required, but personally when I have files or pieces of code that I need to manually keep in sync, I like to have a big comment on **both** files saying that if you edit this file, you need to edit the other file too. (This is the general principle of making it hard to make mistakes :) )